### ISSUE TITLE
--fonts-dir doesn't work

### ISSUE THREAD

**Issue by Snugug** [Compass/compass#881]
Fonts directory option in compass create, as well as config.rb, doesn't appear to work.

``` bash

compass create test --fonts-dir "fonts"
Error: invalid option: --fonts-dir

Usage: compass create path/to/project [options]

Description:
Create a new compass project at the path specified.

Options:
        --using PATTERN              A framework's pattern to use when creating the project.
    -x, --syntax SYNTAX              Specify the syntax to use when generating stylesheets.
                                     One of sass or scss. Defaults to scss.
        --prepare                    Prepare the project by only creating configuration files.
    -r, --require LIBRARY            Require the given ruby LIBRARY before running commands.
                                       This is used to access compass plugins without having a
                                       project configuration file.
    -l, --load FRAMEWORK_DIR         Load the framework or extensions found in the FRAMEWORK directory.
    -L, --load-all FRAMEWORKS_DIR    Load all the frameworks or extensions found in the FRAMEWORKS_DIR directory.
    -q, --quiet                      Quiet mode.
        --trace                      Show a full stacktrace on error
        --force                      Allows some failing commands to succeed instead.
        --dry-run                    Dry Run. Tells you what it plans to do.
        --boring                     Turn off colorized output.
    -c, --config CONFIG_FILE         Specify the location of the configuration file explicitly.
        --app APP                    Tell compass what kind of application it is integrating with. E.g. rails
        --sass-dir SRC_DIR           The source directory where you keep your sass stylesheets.
        --css-dir CSS_DIR            The target directory where you keep your css stylesheets.
        --images-dir IMAGES_DIR      The directory where you keep your images.
        --javascripts-dir JS_DIR     The directory where you keep your javascripts.
    -e, --environment ENV            Use sensible defaults for your current environment.
                                       One of: development (default), production
    -s, --output-style STYLE         Select a CSS output mode.
                                       One of: nested, expanded, compact, compressed
        --relative-assets            Make compass asset helpers generate relative urls to assets.
        --no-line-comments           Disable line comments.
        --bare                       Don't generate any Sass or CSS files.
    -?, -h, --help                   Show this message
```

**Comment by chriseppstein** [comment]
closed by adf0b896a5430a399d18e6378f81455f32495995

**Comment by island205** [comment]
for you with same problem, use `compass v0.1.2.2`
